raspimouse/src/raspimouse_component.cpp

```diff
@@ -185,6 +185,9 @@ CallbackReturn Raspimouse::on_configure(const rclcpp_lifecycle::State &)
   declare_parameter(use_light_sensors_param, true);
   declare_parameter(odometry_scale_left_wheel_param, 1.0);
   declare_parameter(odometry_scale_right_wheel_param, 1.0);
+  declare_parameter(WHEEL_DIAMETER_PARAM, 1.0);
+  declare_parameter(WHEEL_BASE_PARAM, 1.0);
+  declare_parameter(PULSES_PER_REVOLUTION_PARAM, 1.0);
```

Comment:
パラメータの初期値が変わっていたので修正お願いします。

```suggestion
  declare_parameter(WHEEL_DIAMETER_PARAM, 0.048);
  declare_parameter(WHEEL_BASE_PARAM, 0.09);
  declare_parameter(PULSES_PER_REVOLUTION_PARAM, 400.0);
```